htdocs/admin/expedition.php

```diff
@@ -441,7 +441,7 @@
 print $form->textwithpicto($langs->trans("FreeLegalTextOnShippings"), $langs->trans("AddCRIfTooLong").'<br><br>'.$htmltext, 1, 'help', '', 0, 2, 'freetexttooltip').'<br>';
 $variablename = 'SHIPPING_FREE_TEXT';
 if (empty($conf->global->PDF_ALLOW_HTML_FOR_FREE_TEXT)) {
-	print '<textarea name="'.$variablename.'" class="flat" cols="120">'.$conf->global->$variablename.'</textarea>';
+	print '<textarea name="'.$variablename.'" class="flat" cols="120">'.!empty($conf->global->$variablename)?$conf->global->$variablename:"".'</textarea>';
 } else {
```

Comment:
you can use `getDolGlobalString($variablename)` instead `!empty($conf->global->$variablename)?$conf->global->$variablename:""`

Reply:
it's the same thing